### ISSUE TITLE
Non potion consumables sharing CD's (felwood "pots", runes, thistle tea etc)

### ISSUE THREAD

**Issue by knasen88** [LightsHope/issues#826]
https://youtu.be/UW5mv_e4ZmM?t=8m41s
clearly see hes using whipper and night dragon at independent cooldowns, aswell as only 1 minute cooldown instead of the 2 minute that is on this server.

Never seen any of them ever mentioned in any patch notes.

**Comment by Omarthegreatone** [comment]
Yes, seems like they dont share cds with night dragons and such also, very interesting.

**Comment by cudderi** [comment]
It was changed in patch 1.11, excerpt from http://wowwiki.wikia.com/wiki/Patch_1.11 :
"We have re-evaluated the mechanics of consumable items in the game and concluded that these should work in a more intuitive manner. As such, most items that can not be equipped with right click abilities have been streamlined into one of three categories. Using an item of a particular category will trigger a shared cooldown among all other items in the same category. The categories break down as follows with category cooldowns as listed.
Potions 2 minutes: This includes items such as Health Potion, Mana Potion, Invisibility Potion and Mighty Rage Potion.
Aggressive 1 minute: This includes items such as explosives, Really Sticky Glue and Discombobulator Ray.
Non-Aggressive 2 minutes: This includes items such as Healthstone, Night Dragon's Breath, Whipper Root Tuber and Target Dummy."
In the video, the mage also uses two trinkets at the same time, which was patched in 1.10

**Comment by knasen88** [comment]
Welp, well then it's an item progression issue if they want to meddle with it.
